### DISCUSSION TITLE
Donations?

### DISCUSSION THREAD

**Discussion by SamsiFPV** [bloombloombloom/Bloom#84] [General]
Love your work on this, and would love to help, but I am not a C++ dev.
Any chance you have a paypal or something to give you a donation?

**Comment by navnavnav** [comment]
Hey, thanks for this! It's great to see that Bloom is helping others.

RE donations, I don't have any means to accept them ATM, but I may set up something in the near future, given that I'm currently out of a job. I will definitely post an update on here if I do this. Thank you for offering!

**Comment by navnavnav** [comment]
I've just setup a GitHub Sponsors profile: https://github.com/sponsors/navnavnav

I will add a "Sponsor" button to the Bloom repository when I push the next release.

### To anyone thinking about donating/sponsoring:

Please do not feel any pressure to do so. I love working on Bloom. It's challenging and fun. There are times when it stops being fun, but then I take a little break and it becomes fun again! If Bloom helped you, that's great! But that doesn't mean you should be subject to any pressure to donate. Just knowing that it helped you is enough!

If you do still wish to donate, contributions of any size will be appreciated.

Thanks!
